### ISSUE TITLE
remove easy_tello dependency

### ISSUE THREAD

**Issue by azbones** [Fireline-Science/tello_sim#9]
Given easy_tello is fairly simple, it might be better to just rewrite the API to the drone rather than use easy_tello.

**Comment by azbones** [comment]
I just noticed that easy_tello has archived their project, so we should remove this dependency if we get a chance. We should also link it to the DJI API somehow.

**Comment by dullahgtt** [comment]
Are you looking to connect it to the djitellopy python library instead of the easy_tello library? That would easily connect to the drone's command system. I believe I could get that connected pretty easily.

**Comment by azbones** [comment]
We've been a bit caught up in some other development, so haven't looked into djitellopy yet. If you want to open a pull request with that change, we could do some testing with the drone we have and then merge it in. @sean-schaefer any other thoughts?

Also, thanks so much for offering! I've continued to use this with some of the STEM outreach we do and I think it really helps students better understand the process compared to just sending commands to the drone...

**Comment by dullahgtt** [comment]
Yeah, I can definitely do that! I am happy to help. This simulator was actually super helpful as I am doing drone research for pathing and object avoidance and needed a python simulator for testing prior to real world testing. I should have it modified in the next week or so!

**Comment by sean-schaefer** [comment]
djitellopy looks like it would be a great replacement. Thanks for your help @dullahgtt and let me know if you have any questions!

**Comment by dullahgtt** [comment]
So, I figured out all that needs to be done is replace the easytello library with the djitellopy library in the simulator.py file. All the components are the same from the looks of it and it runs perfectly.
